### PR TITLE
change package name to rmt-server #4599

### PR DESCRIPTION
**Pull request by digitaltom**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -22,6 +22,7 @@ package/*.tar.bz2
 package/.osc
 package/rmt.8.gz
 package/_link
+package-ibs
```
Comment (ikapelyukhin) [inline .gitignore]: Maybe that'd be better put into a local .gitignore file?

File: package/rmt-server-rpmlintrc [hunk 2]
```diff
@@ -1,12 +1,9 @@
...
+addFilter("hidden-file-or-dir /usr/share/rmt/.bundle")
 addFilter("hidden-file-or-dir /usr/lib64/rmt/vendor/bundle/*")
 addFilter("unexpanded-macro /usr/lib64/rmt/vendor/bundle/*")
 addFilter("zero-length /usr/lib64/rmt/vendor/bundle/*")
-addFilter("wrong-script-end-of-line-encoding /usr/lib64/rmt/vendor/bundle/*")
-addFilter("hidden-file-or-dir /usr/lib64/rmt/.bundle")
 addFilter("file-not-in-%lang /usr/lib64/rmt/vendor/bundle/*")
 addFilter("no-soname /usr/lib64/rmt/vendor/bundle/*")
 addFilter("non-executable-script /usr/lib64/rmt/vendor/bundle/*")
-addFilter("E: env-script-interpreter .* /usr/lib64/rmt/vendor/bundle/*")
```
Comment (ikapelyukhin) [inline package/rmt-server-rpmlintrc]: Is this warning really gone?

Comment (digitaltom) [reply package/rmt-server-rpmlintrc]: yes, this rpmlin is already pushed to: https://build.opensuse.org/package/show/systemsmanagement:SCC:RMT/rmt-server

Comment (ikapelyukhin) [reply package/rmt-server-rpmlintrc]: Cool :+1:

**Pull request merged**